docs/reference/modules/plugins.asciidoc

```diff
@@ -14,19 +14,26 @@ and more.
 ==== Installing plugins
 
 Installing plugins can either be done manually by placing them under the
-`plugins` directory, or using the `plugin` script. Several plugins can
-be found under the https://github.com/elasticsearch[elasticsearch]
-organization in GitHub, starting with `elasticsearch-`.
+`plugins` directory, or using the `plugin` script.
 
 Installing plugins typically take the following form:
 
 [source,shell]
 -----------------------------------
+bin/plugin --install plugin_name
+-----------------------------------
+
+The plugin will be automatically downloaded in this case from `download.elastic.co` download service using the
```

Comment:
Well... I did not try to change the old behavior but you're right. I guess download.elastic.co works as well as download.elasticsearch.org? Need to test it and if it works, I will update the PR accordingly.